collect_app/src/main/java/org/odk/collect/android/tasks/InstanceGoogleSheetsUploader.java

```diff
@@ -504,17 +583,6 @@ private String getElementTitle(AbstractTreeElement element) {
             if (!path.equals(" ") && columnTitles.contains(path.toString())) {
                 if (answers.containsKey(path.toString())) {
                     answer = answers.get(path.toString());
-                    // Check to see if answer is a location, if so, get rid of accuracy
-                    // and altitude, try to match a fairly specific pattern to determine
-                    // if it's a location [-]#.# [-]#.# #.# #.#
-
-                    if (isLocationValid(answer)) {
-                        // get rid of everything after the second space
-                        int firstSpace = answer.indexOf(' ');
-                        int secondSpace = answer.indexOf(" ", firstSpace + 1);
-                        answer = answer.substring(0, secondSpace);
-                        answer = answer.replace(' ', ',');
```

Comment:
Good catch, @mmarciniak90. This replacement was lost and should be brought back, @xsteelej.

Reply:
Yes very good catch @mmarciniak90 & @lognaturel. My mistake - I looked back at what I thought was a sheet that I had generated before my changes, should have looked at the code in Master instead - Apologies!